git -P diff tiledb/CMakeLists.txt tiledb/CMakeLists.txt
diff --git a/tiledb/CMakeLists.txt b/tiledb/CMakeLists.txt
index 70cb650a4..8e2021d07 100644
--- a/tiledb/CMakeLists.txt
+++ b/tiledb/CMakeLists.txt
@@ -1078,8 +1078,7 @@ set(ver_patch ${CMAKE_MATCH_1})
 
 set(VERSION "${ver_major}.${ver_minor}.${ver_patch}")
 
-set(PKGCONF_REQ_PRIV "libmagic pcre2-8 pcre2-posix")
-string(CONCAT PKGCONF_REQ_PRIV ${PKGCONF_REQ_PRIV} " webp imagedec imageenc imageioutil webpdecoder webpdmux webpmux")
+set(PKGCONF_REQ_PRIV "libmagic")
 
 # PKG Config file
 configure_file(
 