diff --git a/cmake/Modules/FindCapnp_EP.cmake b/cmake/Modules/FindCapnp_EP.cmake
index 317c38519..1f5076bb8 100644
--- a/cmake/Modules/FindCapnp_EP.cmake
+++ b/cmake/Modules/FindCapnp_EP.cmake
@@ -80,11 +80,9 @@ if (NOT CAPNP_FOUND)
       set(CONDITIONAL_PATCH "")
     endif()
 
+    message("CapnProto dir for external project: " ${TILEDB_CAPNPROTO_SRC})
     ExternalProject_Add(ep_capnp
-       PREFIX "externals"
-       DOWNLOAD_NAME ep_capnp.${TILEDB_CAPNPROTO_VERSION}.tar.gz
-       URL ${TILEDB_CAPNPROTO_URL}
-       URL_HASH ${TILEDB_CAPNPROTO_HASH_SPEC}
+      SOURCE_DIR ${TILEDB_CAPNPROTO_SRC}
       CMAKE_ARGS
          -DCMAKE_PREFIX_PATH=${TILEDB_EP_INSTALL_PREFIX}
          -DCMAKE_INSTALL_PREFIX=${TILEDB_EP_INSTALL_PREFIX}
diff --git a/cmake/TileDB-Superbuild.cmake b/cmake/TileDB-Superbuild.cmake
index 60006d65a..3cb2209de 100644
--- a/cmake/TileDB-Superbuild.cmake
+++ b/cmake/TileDB-Superbuild.cmake
@@ -58,6 +58,7 @@ set(INHERITED_CMAKE_ARGS
   -DTILEDB_INSTALL_LIBDIR=${TILEDB_INSTALL_LIBDIR}
   -DCMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES}
   -DTILEDB_EXPERIMENTAL_FEATURES=${TILEDB_EXPERIMENTAL_FEATURES}
+  -DTILEDB_CAPNPROTO_SRC=${TILEDB_CAPNPROTO_SRC}
   -DTILEDB_TESTS_AWS_S3_CONFIG=${TILEDB_TESTS_AWS_S3_CONFIG}
   -DTILEDB_TESTS_ENABLE_REST=${TILEDB_TESTS_ENABLE_REST}
   -DTILEDB_VCPKG_POLICY_EMPTY_PACKAGE=${TILEDB_VCPKG_POLICY_EMPTY_PACKAGE}
