diff --git a/cmake/Modules/FindCatch_EP.cmake b/cmake/Modules/FindCatch_EP.cmake
index 97d5aebb4..da2738fc5 100644
--- a/cmake/Modules/FindCatch_EP.cmake
+++ b/cmake/Modules/FindCatch_EP.cmake
@@ -71,9 +71,6 @@ if (NOT CATCH2_FOUND AND TILEDB_SUPERBUILD)
   ExternalProject_Add(ep_catch
     PREFIX "externals"
     # Set download name to avoid collisions with only the version number in the filename
-    DOWNLOAD_NAME ep_catch.zip
-    URL "https://github.com/catchorg/Catch2/archive/v3.1.0.zip"
-    URL_HASH SHA1=b23753594a743feabd4e30f83b31ebb31081092a
     CMAKE_ARGS
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
       -DCMAKE_INSTALL_PREFIX=${TILEDB_EP_INSTALL_PREFIX}
